TaylorFrameworkTests/TemperTests/NSFileManagerTests.swift

```diff
@@ -26,6 +36,18 @@ class NSFileManagerTests: QuickSpec {
                 manager.removeFileAtPath(filePath)
                 expect(manager.fileExists(atPath: filePath)).to(beFalse())
             }
+            
+            it("should't remove file at path if it's not deleatable") {
```

Comment:
There is no such word `deleatable`. You can use `deletable` or `delible`, but I recommend to use  `removable` here as it's less pretentious.